monitor_memory.py

```
@@ -0,0 +1,363 @@
+                "inline": False
+            })
+
+        try:
+            response = requests.post(
+                self.webhook_url,
+                json={"embeds": [embed]},
+                timeout=5
+            )
+            if response.status_code != 204:
+                print(f"❌ Webhook failed: {response.status_code}")
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)